go.mod

```diff
@@ -11,13 +11,11 @@ require (
 	github.com/getkin/kin-openapi v0.14.0
 	github.com/go-chi/chi v4.0.2+incompatible
 	github.com/go-ozzo/ozzo-validation v3.6.0+incompatible
-	github.com/go-sql-driver/mysql v1.4.1 // indirect
+	github.com/go-sql-driver/mysql v1.5.0 // indirect
 	github.com/golang/protobuf v1.3.2
-	github.com/google/uuid v1.1.1 // indirect
 	github.com/gorilla/websocket v1.4.0
-	github.com/konsorten/go-windows-terminal-sequences v1.0.2 // indirect
 	github.com/lib/pq v1.1.1
-	github.com/mattn/go-sqlite3 v1.11.0 // indirect
+	github.com/mattn/go-sqlite3 v1.14.2 // indirect
```

Comment:
And sqllite O_O

Reply:
100% agree i don't know why it is there

Reply:
```sh
➜  go-base git:(feature-prepare-for-v2) go mod why github.com/mattn/go-sqlite3
# github.com/mattn/go-sqlite3
github.com/contiamo/go-base/v2/pkg/data/managers
github.com/Masterminds/squirrel
github.com/Masterminds/squirrel.test
github.com/mattn/go-sqlite3
```

Reply:
very weird that it doesn't shake out the tests